esm_analysis/carbon.py

```diff
@@ -256,27 +256,85 @@ def _check_variables(ds):
     return sensitivity
 
 
+@check_xarray(0)
+def co2_flx_ocean_sensitivities(ds):
+    """Compute sensitivity of oceanic co2_flux to changes in driver variables.
+
+    .. math: co2_flx_ocean = co2trans * (co2atm - co2ocean) * (1 - ice)
+
+    Args:
+        ds (xr.Dataset): containing cmorized variables:
+                         * co2_flx_ocean [kg m-2 s-1]: surface air-sea CO2 flux
+                         * co2ocean[ppm CO2]: partial pressure of CO2 in
+                                              surface sea water
+                         * CO2[ppm]: partial pressure of atmospheric CO2 at the
+                                     surface
+                         * co2trans [10-9 mol s m-4] : transfer velocity of
```

Comment:
I can't find this term anywhere on the standard output website. It looks like for my last paper I knew how it was computed and then calculated the sensitivity that way.

![Screen Shot 2019-09-02 at 5 31 36 PM](https://user-images.githubusercontent.com/8881170/64135628-2a055f00-cd9f-11e9-82cb-3a825a7a2cef.png)

Reply:
Where "pv" is piston velocity

Reply:
`co2trans` is the piston velocity I guess